### ISSUE TITLE
[BUG] reward page responsiveness in mobile phones also in desktop is not good

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Moksh#501]
https://akshitagupta15june.github.io/Moksh/Pomodro_page/reward.html improve responsiveness

**Comment by akshitagupta15june** [comment]
@MetiUmashri ?

**Comment by visparmar** [comment]
@akshitagupta15june please assign it to me..

**Comment by ishitaa-dhingra** [comment]
@akshitagupta15june please assign this issue to me under GSSOC23

**Comment by MetiUmashri** [comment]
Yes i will work, due to network issues i couldn't reply please @akshitagupta15june please assign it to me

**Comment by MetiUmashri** [comment]
> MetiUmashri

Please assign it to me

**Comment by Rakesh9100** [comment]
@MetiUmashri Start working

**Comment by MetiUmashri** [comment]
Improvisation is almost done will try to submit the responsive design tomorrow.
